src/isValidElementType.lua

```diff
@@ -0,0 +1,14 @@
+local Portal = require(script.Parent.Portal)
+local Type = require(script.Parent.Type)
+
+-- Returns true if the provided object can be used by Roact.createElement
+return function(value)
```

Comment:
I'll add more tests and docs on Monday, but here's isValidElementType for consumers who want to see if a value can be passed to Roact.createElement, which is useful for building APIs that can take any component type

Reply:
Maybe that method should be called `isComponent`, since that's all an 'element type' is?

Reply:
I went with `isValidElementType` to match the API in [react-is](https://www.npmjs.com/package/react-is#usage) but I agree that API name is more clear.